<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HangingObjectsView</class>
 <widget class="QWidget" name="HangingObjectsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>54</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>54</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticlLayout">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>30</height>
      </size>
     </property>
     <widget class="EventFilterDoubleSpinBox" name="doubleSpinBoxHeight">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>2</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxEnable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdate">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>-1</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeight">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>8</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventFilterDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">eventfilterdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
